virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -24,12 +24,13 @@ Cluster configuration procedures require `cluster-admin` privileges.
 [id="virt-getting-started"]
 == Getting started
 
-Create a virtual machine::
-* Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[RHEL virtual machine] using the wizard.
-* Create a Windows virtual machine:
+Create a virtual machine:
+
+* Quick create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-quick-creating-vm_virt-create-vms[virtual machine] using the web console.
```

Comment:
\*side-eyes the "Quick create" phrasing* 

I can see why you'd want this phrasing to reflect how it's shown in the UI, though. Maybe put the link on the "Quick create" phrase itself so that it's differentiated from the rest of the sentence?